modules/s3/s3import.py

```diff
@@ -3935,6 +3936,24 @@ def parseACL(_acl):
                             *acls[rulelist[0]])
 
     # -------------------------------------------------------------------------
+    def import_font(self, url):
```

Comment:
Great if we can add support for zipped Fonts?
The original version was 10Mb unzipped, but distributed as a 2Mb .gz
Supporting both .gz & .zip would be ideal.
This could possibly be brought out into a separate function as I suspect there are other usecases.